web/viewer.css

```diff
@@ -197,15 +213,6 @@ body {
   background-color: var(--body-bg-color);
 }
 
-body,
-input,
-button,
-select {
-  font: message-box;
-  outline: none;
-  scrollbar-color: var(--scrollbar-color) var(--scrollbar-bg-color);
```

Comment:
By removing this particular rule from the `body`, a number of scrollbars no longer look correct in the dark-theme.
To reproduce, use the devtools to switch the viewer to theme and compare the `master` branch with with patch for e.g. the *main* scrollbar, the scrollbar in the sidebarViews, and even the scrollbar the secondaryToolbar.

Rather than applying that everywhere where it matters, which might be difficult to maintain, could we perhaps just set this `scrollbar-color` rule globally as before?